recipes/professor/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "Professor" %}
+{% set version = "2.3.4" %}
+{% set tag = name|lower ~ "-" ~ version %}
```

Comment:
If upstream is amenable to it tagging dots instead of dashes will make the bots life easier for detecting updates.

If not, the trick is to define the version in terms of the tag rather than the opposite.

Reply:
(doesn't matter for now, but if there is a new version)